<launch>

    <node pkg="robot_pose_ekf" type="robot_pose_ekf" name="robot_pose_ekf">
        <param name="output_frame" value="odom_combined"/>
        <param name="base_footprint_frame" value="base_footprint"/>
        <param name="freq" value="30.0"/>
        <param name="sensor_timeout" value="1.0"/>  
        <param name="odom_used" value="false"/>
        <param name="imu_used" value="true"/>
        <param name="vo_used" value="true"/>

        <remap from="imu_data" to="imu/data" />
    </node>

    <node pkg="icp_localization" type="icp_ekf" name="icp_ekf" output="screen"/>
        <param name="cov_x" type="float" value="0" />
        <param name="cov_y" type="float" value="0" />
        <param name="cov_z" type="float" value="0" />
        <param name="cov_yaw" type="float" value="0" />
        <param name="cov_pitch" type="float" value="0" />
        <param name="cov_roll" type="float" value="0" />
    </node>
</launch>